### PR TITLE
Add information on how to reference new STF sensors in the robot description

### PR DESCRIPTION
**Pull request by rnzenha-s**
## Proposed changes

Add information on how to reference the new STF tip sensors when loading the robot URDF.
Also, there is a minor change to the FAQ regarding the supported tactile sensors.
 
## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [x] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [ ] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [x] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: docs/user_guide/sd_robot_description.rst [hunk 1]
```diff
@@ -32,15 +32,17 @@ Current allowed configurations are the following:
...
-There are also arguments that define where and which sensors are located on the hand. It allows placement of sensors on tip, mid and proximal parts of the fingers as well as the palm. Argument names: ``tip_sensors``, ``mid_sensors``, ``prox_sensors``, ``palm_sensor``. Currently, only sensors at the fingertips are available. There are three fingertip sensor types: ``pst``/``bt_sp``/``bt_2p``.
-
-+-------------+-----+------------------+
-|             | PST | Syntouch Biotacs |
-|             |     +---------+--------+
-|             |     |    2p   |   sp   |
-+=============+=====+=========+========+
-| tip_sensors | pst |  bt_2p  |  bt_sp |
-+-------------+-----+---------+--------+
+There are also arguments that define where and which sensors are located on the hand. It allows placement of sensors on tip, mid and proximal parts of the fingers as well as the palm. Argument names: ``tip_sensors``, ``mid_sensors``, ``prox_sensors``, ``palm_sensor``. Currently, only sensors at the fingertips are available. There are three fingertip sensor types: ``mst``/``pst``/``bt_sp``/``bt_2p``.
```
Comment (niko-holmes) [inline docs/user_guide/sd_robot_description.rst]:
```suggestion
There are also arguments that define which sensors are located on the hand and where. It allows placement of sensors on tip, mid and proximal parts of the fingers as well as the palm. Argument names: ``tip_sensors``, ``mid_sensors``, ``prox_sensors``, ``palm_sensor``. Currently, only sensors at the fingertips are available. There are three fingertip sensor types: ``mst``/``pst``/``bt_sp``/``bt_2p``.
```

**Review by niko-holmes — COMMENTED** [review]
Minor sentence structure comment

File: docs/user_guide/fc_FAQ.rst [hunk 2]
```diff
@@ -27,7 +27,8 @@ A: The hand is currently not wireless, as it needs both power and data cables at
 
 **Q: How do the tactile sensors work and how do they communicate?**
 
-A: There are two types of tactile sensing for the fingertips:
+A: There are three types of tactile sensing for the fingertips:
+* **STF**: These are are magnetic-based tactile sensors with 17 distributed taxels sensitive to the 3D forces applied to the sensors' surfaces. They can be fitted in the standard fingertips or (slightly bigger) thumb shapes.
```
Comment (dg-shadow) [inline docs/user_guide/fc_FAQ.rst:31]: Are we using stf or mst? Both seem to appear.

File: docs/user_guide/sd_robot_description.rst (hunk 1 above)
Comment (dg-shadow) [inline docs/user_guide/sd_robot_description.rst]: Should we clarify why it says "three" types but lists four. (I.e. are two types of biotacs only one type of sensor? maybe that's fine but we should probably state it.)

Comment (rnzenha-s) [reply docs/user_guide/sd_robot_description.rst]: You're right, let's just say we support 4: [3d1137c](https://github.com/shadow-robot/sr_dexterous_hand_documentation/pull/180/commits/3d1137c5ffa6387f4f4e1d55307dacc61a325132)

**Review by niko-holmes — APPROVED** [review]

File: docs/user_guide/sd_robot_description.rst (hunk 1 above)
Comment (rnzenha-s) [reply docs/user_guide/sd_robot_description.rst]: Thanks, addressed here: [3d1137c](https://github.com/shadow-robot/sr_dexterous_hand_documentation/pull/180/commits/3d1137c5ffa6387f4f4e1d55307dacc61a325132)

**Review by dg-shadow — APPROVED** [review]

**Pull request merged**